packages/pieces/webflow/src/lib/triggers/new-form-submitted.ts

```diff
@@ -0,0 +1,57 @@
+import { webflowCommon, sitesDropdown } from '../common/common';
+import { createTrigger, TriggerStrategy } from '@activepieces/pieces-framework';
+import { getAccessTokenOrThrow } from "@activepieces/pieces-common";
+
+const triggerNameInStore = 'webflow_created_form_submissions_trigger';
+
+export const webflowNewSubmission = createTrigger({
+    name: 'new_submission',
+    displayName: 'New Submission',
+    description: 'Triggers when Webflow Site receives a new submission',
+    props: {
+        authentication: webflowCommon.authentication,
```

Comment:
Can you please add `form_name` you might not be able to fetch it grammatically (not sure).

But It can be optional text field and you filer it inside the run function.